pyomo/core/plugins/transform/deactivate_trivial_constraints.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: UTF-8 -*-
+"""Transformation to deactivate trivial constraints."""
+from pyomo.core.base.constraint import Constraint
+from pyomo.core.kernel.numvalue import value
+from pyomo.core.plugins.transform.hierarchy import IsomorphicTransformation
+from pyomo.util.plugin import alias
+
+__author__ = "Qi Chen <qichen at andrew.cmu.edu>"
```

Comment:
We are currently working out a standard for `__authors__`, however, i suspect you don't want to include your andrew email (one hopes that is not a long-term email!).  If anything, I would expect something like a github id.